### ISSUE TITLE
Clients hands models are visible 

### ISSUE THREAD

**Issue by sousou63** [dxura/darkrp#15]
![image](https://github.com/user-attachments/assets/a174fd5d-377f-4a81-a4e4-d1a357d74ebb)

**Comment by jusvit** [comment]
Yeah this was somewhat intentional, I just found it funny until we can get an appropriate view model or maybe none at all?
